ray_lightning/ray_ddp_sharded.py

```diff
@@ -1,10 +1,27 @@
 from pytorch_lightning.plugins import DDPSpawnShardedPlugin
+from pytorch_lightning.plugins.precision.sharded_native_amp import \
+    ShardedNativeMixedPrecisionPlugin
+from pytorch_lightning.utilities import _FAIRSCALE_AVAILABLE
+
+if _FAIRSCALE_AVAILABLE:
+    from fairscale.optim.grad_scaler import ShardedGradScaler
 
 from ray.util import PublicAPI
 
 from ray_lightning import RayPlugin
 
 
+# C3 linearization of parent classes will do breadth first since both
+# RayPlugin and DDPSpawnShardedPlugin share a common parent of DDPSpawnPlugin
 @PublicAPI(stability="beta")
 class RayShardedPlugin(RayPlugin, DDPSpawnShardedPlugin):
-    pass
+    def execute_remote(self, model, global_rank, queue):
+        self._model = model
```

Comment:
Looks like this gets set in `RayPlugin.execute_remote`, is this needed here as well?